Hybrid/README.md

```diff
@@ -0,0 +1,114 @@
+platforms: python
+author: viananth
+---
+
+# AzureStack Virtual Machines Management Samples - Python
+These samples demonstrate how to perform common management tasks
+with Microsoft AzureStack Virtual Machines
+using the Azure SDK for Python.
+The code provided shows how to do the following:
+
+- Create virtual machines:
+    - Create a Linux virtual machine
+    - Create a Windows virtual machine
+- Update a virtual machine:
+	- Expand a drive
+	- Tag a virtual machine
+	- Attach data disks
+	- Detach data disks
+- Operate a virtual machine:
+    - Start a virtual machine
+    - Stop a virtual machine
+    - Restart a virtual machine
+- List virtual machines
+- Delete a virtual machine
+
+To see the code to perform these operations,
+check out the `run_example()` function in [example.py](example.py).
+Each operation is clearly labeled with a comment and a print function.
+The examples are not necessarily in the order shown in the above list.
+
+
+## Running this sample
+1.  If you don't already have it, [install Python](https://www.python.org/downloads/).
+
+    This sample (and the SDK) is compatible with Python 2.7, 3.3, 3.4, 3.5 and 3.6.
+
+2.  We recommend that you use a [virtual environment](https://docs.python.org/3/tutorial/venv.html)
+    to run this example, but it's not mandatory.
+    Install and initialize the virtual environment with:
+
+    ```
+    pip install virtualenv
+    virtualenv mytestenv
+    cd mytestenv
+    source bin/activate
+    ```
+
+3.  Clone the repository.
+
+    ```
+    git clone https://github.com/Azure-Samples/virtual-machines-python-manage.git
+    ```
+
+4.  Install the dependencies using pip.
+
+    ```
+    cd virtual-machines-python-manage\Hybrid
+    pip install -r requirements.txt
+    ```
+
+5.  Create a [service principal](https://docs.microsoft.com/en-us/azure/azure-stack/azure-stack-create-service-principals) to work against AzureStack. Make sure your service principal has [contributor/owner role](https://docs.microsoft.com/en-us/azure/azure-stack/azure-stack-create-service-principals#assign-role-to-service-principal) on your subscription.
+
+6.  Fill in and export these environment variables into your current shell. 
+
+    ```
+    export AZURE_TENANT_ID={your tenant id}
+    export AZURE_CLIENT_ID={your client id}
+    export AZURE_CLIENT_SECRET={your client secret}
+    export AZURE_SUBSCRIPTION_ID={your subscription id}
+    export ARM_ENDPOINT={your AzureStack Resource Manager Endpoint}
+    ```
+
+7.  Note that in order to run this sample, Ubuntu 16.04-LTS and WindowsServer 2012-R2-Datacenter images must be present in AzureStack market place. These can be either [downloaded from Azure](https://docs.microsoft.com/en-us/azure/azure-stack/azure-stack-download-azure-marketplace-item) or [added to Platform Image Repository](https://docs.microsoft.com/en-us/azure/azure-stack/azure-stack-add-vm-image).
+
+
+8. Run the sample.
+
+    ```
+    python unmanaged-disks\example.py
```

Comment:
you shouldprobably remove this folder, since you don't have manageddisk anyway

Reply:
We are getting ManagedDisks in a couple of months, so I would like to keep the options open. :)